Makefile

```diff
@@ -47,7 +47,7 @@ ifeq ($(USE_IMAGE_DIGESTS), true)
 endif
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= quay.io/openstack-k8s-operators/ovs-operator:latest
```

Comment:
I don't think we should default this to the "official" image here. The images for quay.io/openstack-k8s-operators/ovs-operator get built automatically when a PR gets built. when you want to build an image for testing you'd want to push it to your own registry. If someone has permissions to push to quay.io/openstack-k8s-operators/ovs-operator using make docker-push could easy overwrite the latest version with a dev/test build. for other operators we kept it as controller:latest and if you want to create a local image you have to set the IMG. FYI if you set the IMAGENAMESPACE, QUAY_PASSWORD, QUAY_USERNAME, REDHATIO_PASSWORD and REDHATIO_USERNAME as repository secrets in github on the forked repo images get auto built to your quay when you push your change. so you can even automate dev builds.

Reply:
the CONTROLLER_TOOLS_VERSION in line 164 should be v0.9.2 to update the api sub module correct

Reply:
Done